core/pom.xml

```diff
@@ -60,6 +60,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Please move all geotools dependency to the parent pom. See here: https://github.com/apache/incubator-sedona/blob/master/pom.xml#L120

Make sure you use the geotools scope variable for the scope

Reply:
Fixed